docker-compose.yml

```diff
@@ -27,8 +27,8 @@ services:
       context: .
       dockerfile: ./perma_web/Dockerfile
       args:
-        chrome-layer-cache-buster: 120070d7-0ce4-47f9-8afe-5f7f3e443dcc
-    image: perma3:0.129
+        chrome-layer-cache-buster: 1aea77be-cc19-415f-850f-ab84e8435d8d
+    image: perma3:0.130
```

Comment:
Do you want to make this `perma3:0.130`, so we can merge https://github.com/harvard-lil/perma/pull/3025 and then this?

Reply:
Sure, hold one.